### ISSUE TITLE
GCop 406

### ISSUE THREAD

**Issue by marjanjavid** [Geeksltd/GCop#189]
I have installed GCop.All.Common version 1.2.6. I tried GCop 406 as the picture below. But no error is kicked. Also there is the same problem about GCop 440 and 660
![406](https://user-images.githubusercontent.com/20684649/44929383-1da38c00-ad70-11e8-8ca7-2ab24be5736d.PNG)

**Comment by Foroughi** [comment]
The new rules (in this case GCop406) are just in GCop.UAT packages. FYI, from now on, every new rule will be added to that package until Paymon confirms they are working properly.

**Comment by marjanjavid** [comment]
At first I didn't installed GCop.UAT. I test it again with installed GCop.UAT and the result is that it has still the previous problem.

**Comment by Foroughi** [comment]
@reporter The bug is fixed on [Development\c4ced123adf753e6deeddd5a3f19bc26d2e0ef43](https://gitlab.com/Paymon/GCop/tree/c4ced123adf753e6deeddd5a3f19bc26d2e0ef43)

**Comment by marjanjavid** [comment]
@Foroughi , @Karvan This rule is kicked by IDE0044. I was reviewing this rule to view changes, I found that It is covered by IDE warnings. So I think we should delete this rule.

**Comment by marjanjavid** [comment]
@Foroughi yes I am sure. There is a screenshot below that shows we have both IDE004 Message and GCop406 warning in this case.

![capture](https://user-images.githubusercontent.com/20684649/49130572-db93d300-f2e9-11e8-9a1d-1a6c03600482.PNG)

**Comment by marjanjavid** [comment]
Still there is this problem on GCop2.6.0

**Comment by marjanjavid** [comment]
@Karvan Should we show GCop406 while IDE0044 shows a message in this case?
